installers/rpm-code/dnf-download.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set +euo pipefail
+source ~/.bashrc
+set -euo pipefail
+source ./dev-env.sh
+declare f
+sudo dnf makecache
```

Comment:
I was trying to avoid this:

  [SKIPPED] rscode-common-20230829.173819-1.x86_64.rpm: Already downloaded

However, it kind of makes sense in this case to not force a download every time. This allows a refresh so I changed the code in 2ac0988.